README.md

```diff
@@ -1,18 +1,24 @@
-Sonoma introduced video reactions to video calls, on by default.  This payload sets app gestures off by default, per app.
 
-**NOTE: THIS ONLY APPLIES SETTINGS WHEN THE USER LOGS IN**
+## About
+
+macOS Sonoma introduced video Reactions for apps using the camera, which is enabled by default on every app.  This configuration profile disables Reactions by default on a per-app basis.
+
+## Behavior
+
+> [!IMPORTANT]
+> **NOTE: THIS CONFIGURATION PROFILE ONLY APPLIES SETTINGS WHEN THE USER LOGS IN**
 
 * If a user is currently logged in when the profile is delivered, no changes are made to any app settings.
 * If a user logs out and logs back in, the settings are applied.
-* If a user turns on gestures for an application in the management scope, the gestures stay on during that login session.  If a user logs out and logs back in, the settings are re-applied. (i.e. if you turn on gestures for Zoom, log out and back in, Zoom gestures are off again)
+* If a user turns on Reactions for an application in the management scope, Reactions will stay on during that login session.  If a user logs out and logs back in, the settings are re-applied (e.g. if you turn on gestures for Zoom, log out and back in, Zoom Reactions are off again).
 
-If you notice an app has gesture settings but is not present in this file (the green "Facetime" icon appears in the menu bar when the camera is accessed by the app), please contribute to benefit others that may have that application as well.
+## Contributing
 
-**To contribute:**
-1) Run `sudo plutil -p /Users/[username]/Library/Group\ Containers/group.com.apple.secure-control-center-preferences/Library/Preferences/group.com.apple.secure-control-center-preferences.av.plist | grep -i reactions-enabled` to see which apps have reactions enabled. You can also run with `grep -i gestures-enabled` to see if any apps have that key set. *NOTE: This needs to be run with `sudo` since the plist is readable only by root.*
-2) Extract the app entry (e.g. `videoeffects/us-zoom-xos/reactions-enabled`)
-3) Fork this repo, update the mobileconfig adding the appropriate line(s) and a boolean key of `false`. Note that you will need to add keys for both "gestures-enabled" and "reactions-disabled" for each application
-4) PR it back.  In the PR make sure to list what the application is in case it's not clear in the mobileconfig setting.
+If you notice an app uses the camera but is not present in this file, please contribute to benefit others who may have that application as well.
 
-   
+1) Launch the target app on your Mac, activate the camera, click the Video button <img src="macos-video-button.png" alt="macOS Video button icon, depicting a green rectangle with a white camera icon in the middle, similar to the FaceTime app icon." height="15" /> in the menu bar, then disable Reactions for that app (see [Use Reactions, Presenter Overlay, and other effects when videoconferencing on Mac](https://support.apple.com/en-us/105117)).
```

Comment:
Are you sure we want to launch app, **disable** reactions, then grep a file for **reactions-enabled?**

Reply:
Yeah, plist keys tend to have a consistent label with a value that changes depending on the user's defined preferences. In this case, the keys are named `reactions-enabled` and `gestures-enabled` and they have values of either true (`1`) or false (`0`).

In the case of this README, whether the setting is disabled isn't actually important (the instructions are just looking for the app identifier), but you need to explicitly launch the app and choose a setting or else the app won't necessarily be listed in these preferences, since macOS has a default setting it applies when the user hasn't defined one and it isn't generally hardcoded into user preferences.